openedx/features/wikimedia_features/wikimedia_general/api/v0/views.py

```diff
@@ -0,0 +1,33 @@
+"""
+Views for wikimedia_general v0 API(s)
+"""
+from rest_framework import generics, status, permissions
+from rest_framework.response import Response
+from opaque_keys.edx.keys import CourseKey
+
+from lms.djangoapps.courseware.courses import get_course_by_id
+
+
+class GetCourseFont(generics.RetrieveAPIView):
```

Comment:
Similarly, we can change this to RetrieveWikiMetaData(generics.RetrieveAPIView):